### PR TITLE
feat: Prepend token symbol

### LINKED ISSUE #396: Prepend token symbol
### Please describe the problem that your request should resolve.
Token related values could be more UX friendly

### Describe the solution you'd like
image

### Additional context
Implement all over the app

### PR DESCRIPTION
**Pull request by janmichek**
## Description
resolves #396 

## Demo

https://github.com/aeternity/aescan/assets/15363559/250803b8-0d3b-4f2d-8ed8-4faddeed4a0a

## Checklist:

- [x] I have read and followed the [Contributing Guide](../../CONTRIBUTING.md)  
- [x] add price label component and reuse over app

### CODE REVIEW

**Comment by janmichek** [comment]
As a followup I created one more refactor issue https://github.com/aeternity/aescan/issues/639

**Comment by janmichek** [comment]
> I found that token symbol is not displayed in all places, for example for this account: ak_xi7SnhwezGZR3PFdX4hHCkEdCBzeKTzqP5V77ybTpoMvraaVG
> 
> ae1
> 2024-01-03.12.52.39.mov
> 
> is this expected @janmichek ?

That's a good discovery. Found that and fixed.

Forgot to mention that the thing you should check is the whole app in A/B comparison of values. Compared to this https://aescan.dev.service.aepps.com

**Comment by janmichek** [comment]
@Liubov-crypto I am not really sure what you wanted to show me. Could you rephrase it?
Does it really differ from the current state of dev? https://aescan.dev.service.aepps.com
What is the account id?

**Comment by Liubov-crypto** [comment]
> @Liubov-crypto I am not really sure what you wanted to show me. Could you rephrase it? Does it really differ from the current state of dev? https://aescan.dev.service.aepps.com What is the account id?

@janmichek I meant additional column for the sent amount. It was different on dev, but now I see the same just with old design.

**Comment by janmichek** [comment]
Dependantly on the order, this needs to be accommodated after rebase https://github.com/aeternity/aescan/pull/700

**Comment by janmichek** [comment]
@michele-franchi can you take a look at this. please?

**Comment by janmichek** [comment]
The build stopped working. Let's continue here https://github.com/aeternity/aescan/pull/809
